Makefile

```diff
@@ -99,3 +99,14 @@ build-image-collector:
 		-t ${REGISTRY}/${COLLECTOR_IMAGE_NAME}:${COLLECTOR_IMAGE_TAG} \
 		-t ${REGISTRY}/${COLLECTOR_IMAGE_NAME}:${COLLECTOR_VERSION} \
 		-f Dockerfile .
+
+build-and-deploy-image-collector:
```

Comment:
I'm torn about this make path and it's contents.  We probably want to tag it as "dev" so something like:
`quay.io/testnetworkfunction/collector:dev` or maybe something based on an environment variable (possible future feature?).